src/rust/bitbox02/src/screen_saver.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Shift Cryptosecurity AG
```

Comment:
please remove this line - since this is a new file the 2023 line alone is good :)

Reply:
Done :)